docs/sources/static/set-up/install/_index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Install Grafana Agent in static mode
+menuTitle: Install static mode
+weight: 100
+aliases:
+- ../set-up/
+- ../
+---
+
+# Install Grafana Agent in static mode
+
+You can install Grafana Agent in static mode on Docker, Kubernetes, Linux, macOS, or Windows.
+
+The following architectures are supported:
+
+- Linux: AMD64, ARM64, ARMv6, ARMv7
+- Windows: AMD64
+- macOS: AMD64 (Intel), ARM64 (Apple Silicon)
+- FreeBSD: AMD64
+
+{{% admonition type="note" %}}
+ppc64le builds are considered secondary release targets and do not have the same level of support and testing as other platforms.
+{{% /admonition %}}
+
+{{< section >}}
+
+{{% admonition type="note" %}}
+Installing Grafana Agent on other operating systems is possible, but is not recommended or supported.
+{{% /admonition %}}
+## Grafana Operator
+
+To get started with Grafana Agent Operator, refer to the Operator-specific [documentation](../../../operator/_index.md).
```

Comment:
Should static mode be linking out to this?

Also: we started referring to the operator as "static mode Kubernetes operator," this might be an outdated usage of the previous term here.

Reply:
This is from the original Static docs. I can remove it... or rename it. I'll remove it fro now and if we need to we can add it back in in another iteration.